### PR TITLE
Issue #SB-19808 Feat: Unit Portal Migration

### PR DESCRIPTION
**Pull request by sharathkashyap**

### CODE REVIEW

File: src/app/client/src/app/modules/core/components/main-menu/main-menu.component.scss [hunk 1]
```diff
@@ -76,4 +76,62 @@
...
+    display:flex;
+    align-items: center;
+    &.active, &:hover {
+      color: var(--white);
+    }
+    &:hover {
+      text-decoration: none;
+    }
+    &--textbook {
+      color: #E7800B;
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/core/components/main-menu/main-menu.component.scss:101]: color is hardcoded

File: src/app/client/src/app/modules/core/components/main-menu/main-menu.component.scss [hunk 2]
```diff
@@ -76,4 +76,62 @@
...
+    &--stories {
+      color: #0531FF;
+      &:hover {
+        background-color: #0531FF;
+      }
+    }
+    &--poem {
+      color: #B620E0;
+      &:hover {
+        background-color: #B620E0;
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/core/components/main-menu/main-menu.component.scss:133]: same as above need changes

File: src/app/client/src/app/modules/content-search/components/search-filter/search-filter.component.scss [hunk 3]
```diff
@@ -104,3 +104,71 @@
...
+        }
+        .sb-pills-container {
+          display: flex !important;
+          flex-wrap: wrap;
+          justify-content: space-between;
+          .pill {
+            padding: calculateRem(8px) calculateRem(16px) !important;
+            color: #6236FF !important;
+            margin: 0 0 calculateRem(8px) 0 !important;
+            background-color: rgba(98,54,255,0.13) !important;
```
[the remark this answers is not in the document]
Comment (rajeevsathish) [reply src/app/client/src/app/modules/content-search/components/search-filter/search-filter.component.scss:139]: Resolved

**Comment by rjshrjndrn** [comment]
@harshavardhanc there seems some sonar cloud issue. Please check

**Pull request merged**